@3rdweb-sdk/react/hooks/usePayments.ts

```diff
@@ -862,3 +889,181 @@ export function usePaymentsSellerByAccountId(accountId: string) {
     { enabled: !!paymentsSellerId && !!address },
   );
 }
+
+export type PaymentsWebhooksType = {
+  id: string;
+  sellerId: string;
+  url: string;
+  isProduction: boolean;
+  createdAt: Date;
+};
+
+export function usePaymentsWebhooksByAccountId(accountId: string) {
+  invariant(accountId, "accountId is required");
+
+  const address = useAddress();
+  const { paymentsSellerId } = useApiAuthToken();
+  const [getWebhooksBySellerId] = useWebhooksBySellerIdLazyQuery();
+
+  const queryInfo = useQuery(
+    paymentsKeys.webhooks(accountId),
+    async () => {
+      invariant(paymentsSellerId, "no payments seller id found");
+      const { data, error } = await getWebhooksBySellerId({
+        variables: {
+          sellerId: paymentsSellerId,
+        } as WebhooksBySellerIdQueryVariables,
+      });
+
+      if (error) {
+        console.error(error);
+      }
+
+      if (data && data.webhook.length > 0) {
+        return data.webhook.map((webhook) => ({
+          id: webhook.id,
+          sellerId: webhook.seller_id,
+          url: webhook.url,
+          isProduction: webhook.is_production,
+          createdAt: new Date(webhook.created_at),
+        })) as PaymentsWebhooksType[];
+      } else {
+        return [] as PaymentsWebhooksType[];
+      }
+    },
+    { enabled: !!paymentsSellerId && !!address },
```

Comment:
```suggestion
    { enabled: !!paymentsSellerId && !!accountId },
```

We should refetch on accountId change so we don't even have to get the address